internal/service/event_post_processing.go

```diff
@@ -682,26 +650,6 @@ func (s *eventPostProcessingService) prepareProcessedEvents(ctx context.Context,
 	return results, nil
 }
 
-// isSupportedAggregationType checks if the aggregation type is supported for post-processing
-func (s *eventPostProcessingService) isSupportedAggregationType(agg types.AggregationType) bool {
-	return agg == types.AggregationCount || agg == types.AggregationSum
-}
-
-// isSupportedBillingModel checks if the billing model is supported for post-processing
-func (s *eventPostProcessingService) isSupportedBillingModel(billingModel types.BillingModel) bool {
-	// We support usage-based billing models
-	// FLAT_FEE is not appropriate for usage-based billing as it doesn't depend on consumption
-	return billingModel == types.BILLING_MODEL_FLAT_FEE
-}
-
-// isSupportedAggregationForPostProcessing checks if the aggregation type and billing model are supported
-func (s *eventPostProcessingService) isSupportedAggregationForPostProcessing(
-	agg types.AggregationType,
-	billingModel types.BillingModel,
-) bool {
-	return s.isSupportedAggregationType(agg) && s.isSupportedBillingModel(billingModel)
-}
-
 // Find matching prices for an event based on meter configuration and filters
 func (s *eventPostProcessingService) findMatchingPricesForEvent(
```

Comment:
we need to bake in the concept of subscription line item start end time check against event timestamp